lib/widgets/content.dart

```diff
@@ -813,27 +812,26 @@ class UserMention extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      decoration: _kDecoration,
+      decoration: BoxDecoration(
+        // TODO(#646) different background between direct and wildcard mentions
+        color: const HSLColor.fromAHSL(0.2, 240, 0.7, 0.7).toColor(),
+        borderRadius: const BorderRadius.all(Radius.circular(3))),
       padding: const EdgeInsets.symmetric(horizontal: 0.2 * kBaseFontSize),
       child: InlineContent(
         // If an @-mention is inside a link, let the @-mention override it.
         recognizer: null,  // TODO make @-mentions tappable, for info on user
         // One hopes an @-mention can't contain an embedded link.
         // (The parser on creating a UserMentionNode has a TODO to check that.)
         linkRecognizers: null,
+
+        // TODO(#647) when self-user is non-silently mentioned, make bold, and:
+        // TODO(#646) when self-user is non-silently mentioned,
+        //   differ font color between direct and wildcard mentions
```

Comment:
nit:
```suggestion
        // TODO(#646) when self-user is non-silently mentioned,
        //   distinguish font color between direct and wildcard mentions
```

At least for me, "differ" is only an intransitive verb.